src/Effect/Console.js

```diff
@@ -1,55 +1,17 @@
 "use strict";
 
-exports.log = function (s) {
-  return function () {
-    console.log(s);
-    return {};
```

Comment:
`return {}` seems unnecessary, It's the same as returning `undefined` (which is what these console functions return anyway), and even if something else is returned, PS will ignore it.